Add sparql-playground

paul121 opened the pull request:
Some simple docs, utility scripts and file/directory structure to start playing with our data triple store + sparql queries...

see `sparql-playground/README.md`

paul121 commented:
Update the PR description to describe high-level changes

I've also added a Github status check which adds an easy link to navigate to the yasgui SPARQL query for the current dataset. Available at bottom of PR

<img width="1970" height="128" alt="click-to-explore" src="https://github.com/user-attachments/assets/68a70365-d7f5-48cf-9bdb-a55ab58dd85b" />

S4mmyb reviewed (commented):
LGTM - we discussed removing the C01 example files and updates to ProjectInfo which I'll handle in my pr #43

paul121 commented:
Okay!
- Rebased on main
- Removed temp data
- Removed some minor changes to project schemas required for validation that @S4mmyb will add back in #43
